SU-SWS/vpue_undergrad_subtheme
# UE-301 - Inline headings and Big Red list

# READY FOR REVIEW

# Summary
- Styles for the WYSIWYG
   - inline heading
   - big red numbered list

# Review By (Date)
- Soon

# Criticality
- Med
- VPUE Subtheme

# Urgency
- Med

# Review Tasks

## Setup tasks and/or behavior to test

1. Check out this branch
2. Rebuild Cache and import config `drush cr ; drush ci`
3. Navigate to a Basic page and create a text area.
4. Test the inline heading:
5. Add an image with position none.
5. Add your heading
4. Edit the source and wrap the heading and image in a div with the class `inline-heading`. 
5. Try both the image and the heading first
6. add other text and see how it flows.
4. Verify the heading it next to the image and top aligned with spacing between the two.
5. Test the Big Red Numbered List:
6. Add a `<ol>` and put in content
7. Edit the source and add this class to the `ol` : `class="big-red-list"`

### Site Configuration Sync

- Is there a config:export in this PR that changes the config sync directory?

## Front End Validation

## Backend / Functional Validation
### Code

### Code security

## General

# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
- [UE-301](https://stanfordits.atlassian.net/browse/UE-301)

# Resources
- [AMP Tool](https://stanford.levelaccess.net/index.php)
- [Accessibility Manual Test Script](https://docs.google.com/document/d/1ZXJ9RIUNXsS674ow9j3qJ2g1OAkCjmqMXl0Gs8XHEPQ/edit?usp=sharing)
- [HTML Validator](https://validator.w3.org/)
- [Browserstack](https://live.browserstack.com/dashboard) and link to [Browserstack Credentials](https://asconfluence.stanford.edu/confluence/display/SWS/External+Account+Credentials)
